lib/modules/manager/conan/artifacts.ts

```diff
@@ -61,7 +73,12 @@ export async function updateArtifacts(
     await writeLocalFile(packageFileName, newPackageFileContent);
 
     logger.trace(`Updating ${lockFileName}`);
-    await conanLockUpdate(packageFileName, isLockFileMaintenance);
+    await conanLockUpdate(
+      packageFileName,
+      isLockFileMaintenance,
+      config.constraints?.conan,
+      config.constraints?.python,
```

Comment:
is there a way to detect it from repo? for a first merge it's ok as is if not easily possible.

Reply:
Afaik with conan you cannot define the python or conan version you are using, you are bound to the python and conan version that is available on the system.
So the only way for us to support conan version or the underlaying python version is by using constraints.

However, I am not that familiar with conan, so it could be that repo extraction is possible and we add it at a later stage as you said.